README.md

```diff
@@ -1,125 +0,0 @@
-# Parallel Transform Stream
-
-A NodeJS transform stream which runs transformations in parallel and preserves input order.
-
-```sh
-npm install parallel-transform-stream --save
```

Comment:
A prefix? You mean, as in `node-parallel-transform-stream`? I think having a `node-` prefix in the github repo makes sense, as not all modules in [github.com/ubilabs](github.com/ubilabs) are node modules. Having a `node-` prefix as package name for the **Node** Package Manager feels a bit redundant...